larray/inout/xw_reporting.py

```diff
@@ -242,6 +242,18 @@ def add_graph(self, data, title=None, template=None, width=None, height=None):
         height : int, optional
             height of the title item. The current default value is used if None
             (see :py:obj:`~ExcelReport.set_item_default_size`). Defaults to None.
+        **kwargs
+
+          * min_y: int, optional
+              minimum value for the Y axis.
+          * max_y: int, optional
+              maximum value for the Y axis.
+          * periodicity: int, optional
+              periodicity along the X axis (i.e. spacing value between two ticks).
+          * func: function, optional
```

Comment:
1. It's a back door implemented in case users have "urgent" requests. I do not expect users to define their own function but somebody from the CIC team. That's why I didn't mention it in the changelog
2. This function `func` is called inside the `dump()` method which is called inside the `to_excel()` method. So there is no interest to return a Chart object for the moment. Maybe after the refactoring of the `ExcelReport` and `ReportSheet` objects (see issue #900). But I think this refactoring will not come very soon.

Reply:
I like the "backdoor" description 😄